test/client/fixtures/core/page-unload-barrier-test.js

```diff
@@ -101,4 +102,52 @@ $(document).ready(function () {
                 start();
             });
     });
+
+    module('Regression');
+
+    if (browserUtils.isIE9 || browserUtils.isIE10) {
+        asyncTest("Should resolve waiting promise if click on a link with javascript, which doesn't lead to unload (GH-608)", function () {
```

Comment:
I see some unnecessary words here. Should be something like 'Should resolve promise if click on a link doesn't lead to page unload (GH-608)'
/cc @VasilyStrelyaev